h/models/document.py

```diff
@@ -18,10 +18,13 @@
 log = logging.getLogger(__name__)
 
 
-class ConcurrentUpdateError(transaction.interfaces.TransientError):
+class ConcurrentUpdateError(Exception):
     """Raised when concurrent updates to document data conflict."""
 
 
+mark_error_retryable(ConcurrentUpdateError)
```

Comment:
I find it a little weird that this is hanging out here, don't really have any brilliant alternatives. I suppose you could make a special register_exceptions_as_retryable in a more pyramid specific area rather than where the exception is defined but maybe it's better to keep those close to the exception definition.

Reply:
:man_shrugging: It's just how pyramid_retry is supposed to be used. You could move it into an includeme. But when I've seen this used in other projects it was just at module level